shared/team-building/email-input/index.tsx

```diff
@@ -1,36 +1,37 @@
 import * as React from 'react'
-import {debounce} from 'lodash-es'
 import * as Container from '../../util/container'
 import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
 import * as TeamBuildingGen from '../../actions/team-building-gen'
+import * as Constants from '../../constants/team-building'
+import * as Types from '../../constants/types/team-building'
 import {AllowedNamespace} from '../../constants/types/team-building'
 import {validateEmailAddress} from '../../util/email-address'
 import ContinueButton from '../continue-button'
 
 type EmailInputProps = {
   namespace: AllowedNamespace
+  search: (query: string, service: Types.ServiceIdWithContact) => void
```

Comment:
🐑 
```suggestion
  search: (query: string, service: 'email') => void
```

A small guarantee that this code will ever only search emails

Reply:
SGTM, I'll add it to the part 3 PR. Thanks for looking at these!